GVRf/Framework/jni/objects/post_effect_data.h

```diff
@@ -160,7 +160,7 @@ class PostEffectData: public HybridObject {
 
 private:
     ShaderType shader_type_;
-    std::map<std::string, std::shared_ptr<Texture>> textures_;
+    std::map<std::string, Texture*> textures_;
```

Comment:
similar here... Actually based on this patch, getTexture and the map textures_ will never be reached. Later after this patch get merged, some one else or  I will make a clean-up patch for these.